README.md

```diff
@@ -1,3 +1,7 @@
+### About This Repository
+
+The contents of this repository complement the [Chef at PagerDuty](https://www.pagerduty.com/blog/chef-at-pagerduty/) article on PagerDuty blog. The article was published in November 2013 and the repository represents our thinking and Chef workflow at the time; it has not been updated since, and does not necessarily reflect our current processes.
```

Comment:
I like the bit that explains why this repo exists.

The bit that explains that this no longer reflects the way we do things is a little bit hidden, and I would pull it out into it's own section above everything else, with some ⚠️  emojis alongside it perhaps.

We should also make use of [this feature](https://github.com/blog/2460-archiving-repositories) here.